zephyr/src/rt/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+if(CONFIG_NXP_MONOLITHIC_WIFI)
+	set(hal_blobs_dir ${ZEPHYR_HAL_NXP_MODULE_DIR}/zephyr/blobs)
```

Comment:
Similar to the `rw61x` this should be 
`set(hal_blobs_dir ${ZEPHYR_HAL_NXP_MODULE_DIR}/zephyr/blobs/rt)`